packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -198,15 +196,18 @@ export function endpointPosAndSide(
 	};
 }
 
-function toSequencePlace(pos: number | "start" | "end", side: Side): SequencePlace {
-	return typeof pos === "number" ? { pos, side } : pos;
+export function toSequencePlace(
```

Comment:
these are reused for apply stashed op, so need to be exported, and support undefined sides